package.json

```diff
@@ -4,7 +4,7 @@
   "homepage": "https://github.com/fabiosantoscode/terser",
   "author": "Mihai Bazon <mihai.bazon@gmail.com> (http://lisperator.net/)",
   "license": "BSD-2-Clause",
-  "version": "3.17.0",
+  "version": "3.17.1",
```

Comment:
I believe that this line change should be done by @fabiosantoscode

Reply:
ok, i'll change that